modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingRenderView.js

```diff
@@ -1387,6 +1438,14 @@ export default function ChangeTrackingRenderView({
 						</div>
 
 						<div className="entry-description">{description}</div>
+
+						<div className="entry-status">
+							{Liferay.Language.get('workflow-status')}
+
+							<WorkflowStatusLabel
+								workflowStatus={workflowStatus}
+							/>
+						</div>
```

Comment:
@pei-jung, something like this?
`<div className="entry-description">
	<span>{description}{' '}</span>

	<span>
		{Liferay.Language.get('workflow-status-colon')}{' '}

		<WorkflowStatusLabel
			workflowStatus={workflowStatus}
		/>
	</span>
</div>`

I added a new language key to add the colon and kept the original key as is. I saw that something like this has been done with other language keys, like `warning-colon`

Reply:
I believe you can do it like this so that you don't need to add a new lang key.

```
<div className="entry-description">
	<span>{description} </span>
	
	<span>{Liferay.Language.get('workflow-status')}: </span>
	<WorkflowStatusLabel
		workflowStatus={workflowStatus}
	/>
</div>
```